### ISSUE TITLE
Path differs on Google Cloud vs Emulator

### ISSUE THREAD

**Issue by Tivoli** [GoogleCloudPlatform/cloud-functions-emulator#26]
### Description

The req.path is one level up on the emulator vs a function deployed on GCF.

### Steps to reproduce

```
exports.test = function test(req, res) {
  const url = req.path;
  res.json({url});
}
```

The url will be '/test' when using the emulator and on GCF the url will be '/'.

Thanks!

**Comment by jmdobry** [comment]
This is fixed in the unreleased 1.0.0-alpha.1 version on the master branch.

**Comment by jmdobry** [comment]
1.0.0-alpha.1 has been released to NPM, install it with:

    npm install -g @google-cloud/functions-emulator@1.0.0-alpha.1

or

    yarn global add @google-cloud/functions-emulator@1.0.0-alpha.1
